python/ray/serve/tests/test_deploy.py

```diff
@@ -218,7 +226,7 @@ async def __call__(self, **kwargs):
 
     # Now the goal and requests to the new version should complete.
     # We should have two running replicas of the new version.
-    client._wait_for_application_running("app", timeout_s=10)
+    client._wait_for_application_running("app")
```

Comment:
why remove the timeout here?

Reply:
in my local testing i noticed that, it sometime takes more than 10s for the `v2` app to become available. Let's instead rely on pytest timeouts (which by default is set to 180s), which is generous enough.